<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarpEditDialog</class>
 <widget class="QDialog" name="WarpEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Warp Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Area</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="areaBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Map</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="mapBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Spawn</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="spawnBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Areas unlocked on the world map when this warp is used:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="areaBox_1">
           <property name="text">
            <string>1 (Poloy Forest)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="areaBox_2">
           <property name="text">
            <string>2 (Caron Forest)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="areaBox_3">
           <property name="text">
            <string>3 (Volcanic Tunnel)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="areaBox_4">
           <property name="text">
            <string>4 (Polly Mt. 1)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="areaBox_5">
           <property name="text">
            <string>5 (Polly Mt. 2)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="areaBox_6">
           <property name="text">
            <string>6 (Lake Rocky)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="areaBox_7">
           <property name="text">
            <string>7 (Lake Crystal)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="areaBox_8">
           <property name="text">
            <string>8 (Green Island)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="areaBox_9">
           <property name="text">
            <string>9 (Cavern Island)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="areaBox_10">
           <property name="text">
            <string>10 (Coco Island)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="areaBox_11">
           <property name="text">
            <string>11 (Battle Fortress 1)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="areaBox_12">
           <property name="text">
            <string>12 (Battle Fortress 2)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Warp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="warpNumberBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WarpEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WarpEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
